### DISCUSSION TITLE
Allow changes to existing railway stations

### DISCUSSION THREAD

**Discussion by MaximDementyev** [voxeltycoon/issues#482] [General]
**Is your feature request related to a problem? Please describe.**
At the moment there is no possibility to change the railway station. Add/remove path, change length. Now it is necessary to remove the station and build a new one. 

**Describe the solution you'd like**
I want to see the functionality to change the parameters without removing the station

**Describe alternatives you've considered**
allow you to build a new building on top of the old one. from the cost of the new subtract the full cost of the old

**Additional context**

**Comment by katepolak** [comment]
As said before on your other suggestions, please use the template for suggestions, it's there for a reason.

**Comment by kristofeI** [comment]
I confirm it's very tedious not to have the ability to alter already built station. It's the problem, because now it's hard to tell how your train would fit into newly built station (not displaying any numbers considering your trains length vs. station length). Also, it's counter-intuitive to remove whole station just to change its length. Are there any updates on this issue?

**Comment by andrewpey** [comment]
No specific dates yet, but this is definitely on our list of things to do.
